tests/unit/test-cmd/test.run.js

```diff
@@ -127,6 +127,17 @@ describe('run', () => {
     });
   });
 
+  it('passes -jsconsole when --browser-console is specified', () => {
+    const cmd = prepareRun();
+    const {firefoxApp} = cmd.options;
+
+    return cmd.run({browserConsole: true}).then(() => {
+      assert.equal(firefoxApp.run.called, true);
```

Comment:
I am just an observer but I noticed about 13-14 instances of "assert\.equal\(.*,.*true.*\)", in this file alone.  About ~80 locations in just the web-ext/tests directory.  But assert.ok was used in only 1 location before.  is there likewise an assert.nok or !assert.ok for an assertion of false?  Any benefit to using the shortcut besides shorter line?  Any concern over readability of code?  For example, equals true/false seems clear, where assert.nok or !assert.ok() seems confusing.  If a true must later be changed to a false, or vice-versa.